src/functions/tag.js

```diff
@@ -7,17 +7,22 @@ import * as mml from "../buildMathML";
 defineFunctionBuilders({
     type: "tag",
     mathmlBuilder(group, options) {
+        const pad = new mathMLTree.MathNode("mtd", []);
+        pad.setAttribute("width", "50%");
+
         const table = new mathMLTree.MathNode("mtable", [
-            new mathMLTree.MathNode("mlabeledtr", [
+            new mathMLTree.MathNode("mtr", [
+                pad,
                 new mathMLTree.MathNode("mtd", [
-                    mml.buildExpressionRow(group.tag, options),
+                    mml.buildExpressionRow(group.body, options),
                 ]),
+                pad,
```

Comment:
I'm not sure it's safe to use the same instance twice.

Reply:
Okay, I've made a duplicate.